docs/gestor-tareas.md

```diff
@@ -0,0 +1,35 @@
+# Gestor de tareas
+
+## Criterios de búsqueda de un gestor de tareas
+
+Un gestor de tareas es una herramienta necesaria para la ejecución de los tareas como, por ejemplo, la ejecución de test usando las herramientas de construcción, la comprobación de que la sintaxis del código fuente es correcta o la gestión de tareas del lenguaje que se esté usando. Esto nos permite automatizar cualquier tarea de nuestro proyecto en una única orden. Para elegir uno para nuestra aplicación, se establecen los siguientes criterios:
+
+- Que permita gestionar tareas de JavaScript, sobre todo en el entorno de NodeJS, que es el que se utilizará.
+- Que gestione todas las tareas desde un único fichero donde se incluyan todas ellas de manera organizada.
+- Que esté actualizado y tenga un continuo mantenimiento, para que nuestra aplicación no quede obsoleta en un futuro.
+
+## Gestores de tareas a investigar a partir los cuales elegir:
+
+### npm
+
+Viene incorporado en NodeJS y, como ya se ha explicado en #26, funciona tanto como gestor de tareas como gestor de dependencias. Hay dos tipos de tareas con npm, tareas integradas (las que son comunes a la mayoría de los proyectos, como `test` o `install`) y tareas propias (aquellas que nos permite crear npm mediante scripts propios). Sin embargo, no es muy común utilizarlo de esta forma, ya que hay herramientas que son gestores de tareas específicos y mucho más completos y con mas funcionalidades.
+
+### Grunt
+
+Se basa en el entorno en tiempo de ejecución NodeJS de JavaScript. Guarda toda la información en el archivo gruntfile.js, donde se incluye el código original con el que se integran y configuran los módulos de cada una de las tareas. Grunt utiliza principalmente el módulo fs (file system) del entorno de NodeJS Además, Grunt está estrictamente orientado a los archivos, creándolos de forma temporal y local como resultado de las tareas ejecutadas. 
```

Comment:
¿En qué creéis que contribuye este párrafo a que yo evalúe si habéis alcanzado el objetivo o no?